core/debug/loading-inspection-manager.php

```diff
@@ -32,6 +33,16 @@ public function register_inspection( $inspection ) {
 	}
 
 	public function run_inspections() {
+		if ( Utils::get_super_global_value( $_GET, 'post' ) == get_option( 'woocommerce_shop_page_id' ) ) {
```

Comment:
@baghdasarovelementor It seems that you should rather register a new inspection. WDYT?
(Side thought: Can the new inspection be registered in Pro? Because Core cannot edit the Shop page).

Reply:
@louiswol94 I am not sure about Side thought. 

`Ideally, this should be clearer for users why they can’t edit the shop page, and a button to help them get Elementor Pro or launch the Theme Builder to create a new Products Archive Site Part.`

This part is from the task, and while I deactivated Elementor Pro I still can edit the shop page and get messages.